frontend/src/features/admin-form/share/ShareFormModal.tsx

```diff
@@ -183,8 +193,8 @@ export const ShareFormModal = ({
           padding-bottom: 8px;
         "
       >
-        If the form below is not loaded, you can also fill it in at
-        <a href="${shareLink}">here</a>.
+        ${t('embed.fallbackText')}
+        <a href="${shareLink}">${t('embed.here')}</a>.
```

Comment:
nit: would be awesome if you could interpolate the link component here!
this would allow the sentence structure to better fit the grammar of the language translation.

Reply:
Right-o, gotcha. Yup agreed.

1. This interpolation generates the embeddable HTML snippet for an admin to [copy and embed](https://github.com/opengovsg/FormSG/pull/8843/files/664fb872db9047c82c8e69bfbbc3279b042b7632..5ad94afb4f86e3d2970db2ebf368479bedbe7fb4#diff-7991b38254ab74371ea26bef8346647e26d210d7d03c3fb7ec150f3379970adeR358-R363) in their own site.
2. It's not a security issue, but component interpolation can't be used.
3. This string interpolation approach would work just fine.